projects/ng-rule-editor/src/lib/rule-editor.service.spec.ts

```diff
@@ -13,4 +15,48 @@ describe('RuleEditorService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
-});
+
+  it("should return scored questionnaire with var1", () => {
+    expect(service.addTotalScoreRule(mockdata.fhir, mockdata.linkId)).to.include(mockdata.var1);
```

Comment:
It looks to me like mockdata.fhir contains two Questionnaires, "bmi" and "phq9".  Shouldn't you just be passing "phq9" in here?

Reply:
Fixed